naughty/fedora-37/3598-network-manager-addresses-empty

```diff
@@ -0,0 +1,4 @@
+Traceback (most recent call last):
+*
+wait_js_cond(ph_in_text("#networking-interfaces tr[data-interface='cockpit1']","10.111.")): Uncaught (in promise) Error: actual text: cockpit10 bps0 bps
```

Comment:
Actual text "cockpit10" sounds brittle, though. Your [referenced output](https://github.com/cockpit-project/cockpit/pull/17530#issuecomment-1177519797) says "timeout", and let's also explicitly match on "Error: timeout".

Reply:
Ah,  I must have messed up and retested with my local rawhide image?

Reply:
```
+ find /var/lib/systemd/coredump -type f -delete
+ logger -p user.info 'COCKPITTEST: end TestBonding.testBasic'
Killing browser (pid 596559)
killing ssh master process 596472
Traceback (most recent call last):
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/verify/check-networkmanager-bond", line 43, in testBasic
    self.wait_for_iface(iface1)
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/verify/netlib.py", line 163, in wait_for_iface
    raise e
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/verify/netlib.py", line 159, in wait_for_iface
    self.browser.wait_in_text(sel, text)
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/common/testlib.py", line 621, in wait_in_text
    self.wait_js_cond("ph_in_text(%s,%s)" % (jsquote(selector), jsquote(text)),
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/common/testlib.py", line 564, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
  File "/home/jelle/projects/cockpit/rawhide_tuned_workaround/test/common/testlib.py", line 270, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
testlib.Error: timeout
wait_js_cond(ph_in_text("#networking-interfaces tr[data-interface='cockpit1']","10.111.")): Uncaught (in promise) Error: actual text: cockpit10 >
cdp: {'source': 'network', 'level': 'error', 'text': 'Failed to load resource: the server responded with a status of 401 (Authentication failed)>

# Result testBasic (__main__.TestBonding) failed
# 1 TEST FAILED [51s on t14s]
not ok 1 test/verify/check-networkmanager-bond TestBonding.testBasic [ND@0]
```

Soo that's fun

Reply:
Hmpfft seems to cause different naughties so I better work on https://artifacts.dev.testing-farm.io/f77425f6-e8c9-4dda-bf99-0e549e312382/work-allDFXaCv/log.txt